daemon/runtime_unix.go

```diff
@@ -0,0 +1,65 @@
+// +build !windows
+
+package daemon
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/containerd/containerd/runtime/linux/runctypes"
+	v2runcoptions "github.com/containerd/containerd/runtime/v2/runc/options"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/daemon/config"
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+)
+
+const (
+	defaultRuntimeName = "runc"
+	linuxV1Runtime     = "com.docker.runtime.runc.v1"
+	linuxV2Runtime     = "com.docker.runtime.runc.v2"
```

Comment:
Why not just use containerd runtime name?

Reply:
I did this because we provide our own configuration... but maybe it's not needed.

Reply:
FYI this code moved in my last push, but is currently unchanged.

Reply:
Let's just use containerd names to avoid confusion.